core/api/jrpc/jrpc_server_impl.hpp

```diff
@@ -30,11 +30,20 @@ namespace kagome::api {
      * @param request json request string
      * @param cb callback
      */
-    void processData(std::string_view request, const ResponseHandler &cb) override;
+    void processData(std::string_view request,
+                     const ResponseHandler &cb) override;
+
+    /**
+     * @brief creates valid jsonrpc response
```

Comment:
```suggestion
     * @brief creates a valid jsonrpc response and passes it to \arg cb
```
Suggest this because I would be confused by a void method which description says it creates something.

Reply:
agree